open-build-service/debian/copyright

```diff
@@ -0,0 +1,57 @@
+netqmail-1.04
```

Comment:
Can this be a symlink to top-level `COPYRIGHT` so that it's always in sync? (And so that if we ever misformat the main file, OBS's Debian build will barf and we'll know right away.)

Reply:
The build fails if this is a symlink. Just tried it on open build service in my local repo

```
[   19s] Scanning //usr/src/packages/SOURCES.DEB/notqmail_1.09.orig.tar.gz...
[   19s] Generating //usr/src/packages/SOURCES.DEB/notqmail_1.09-1.1+5.1.diff
[   19s] Scanning //usr/src/packages/SOURCES/debian.tar.gz...
[   19s] Archive contains a link: debian/copyright -> ../../COPYRIGHT
[   19s] debian transforming failed.
```